chat.delta.desktop.yml

```diff
@@ -70,6 +72,41 @@ modules:
         url: https://ftp.osuosl.org/pub/blfs/conglomeration/cyrus-sasl/cyrus-sasl-2.1.27.tar.gz
         sha256: 26866b1549b00ffd020f188a43c258017fa1c382b3ddadd8201536f72efb05d5
 
+  - name: deltachat-core-rust
+    buildsystem: simple
+    build-options:
+      env:
+        electron_config_cache: /run/build/delta/npm-cache
```

Comment:
is this needed for rust?

Reply:
nah. I just copy and pasted stuff. I intended to clean the commits up once it's working. Good catch, though!